CIScripts/Common/Invoke-UntilSucceeds.ps1

```diff
@@ -4,10 +4,12 @@
 function Invoke-UntilSucceeds {
     <#
     .SYNOPSIS
-    Utility wrapper for making sure that the assert is consistently true. It works by retrying
-    the assert every Interval seconds, up to Duration.
+    Repeatedly calls a script block until its return value evaluates to true. Subsequent calls
```

Comment:
This description doesn't match the code – it suggests that on falsy result it will retry the command, but actually it throws on the first falsy result.
I'd just remove the `"Did not evaluate to True."` check and logic, leaving it up to the caller.